static/js/jira-select-card-option.js

```diff
@@ -7,7 +7,16 @@ function openChildWindow(url) {
 	const interval = setInterval(function () {
 		if (child.closed) {
 			clearInterval(interval);
-			AP.navigator.reload();
+			if (Boolean($("#redirectToGitHubConfigPage").val())) {
```

Comment:
instead of `Boolean` casting, you can just do double !: `if (!!$("#redirectToGitHubConfigPage").val())`

Reply:
nvm, you don't need the conditional at all.  When the window closes, just go to the github-post-install-page

Reply:
Removing this condition means doing a redirect to the jira config page everytime. Then what about the case when the user doesn't make any connections. If the user simply closes the child window, the url in the parent window will navigate too. Not sure if thats ok, what do you think @rachellerathbone?

Reply:
We need a solution that works for both scenarios:
1. user clicks connect on connect an org screen. Here they need to be redirected to the GH config page. This should happen for both cloud and server customers
2. user clicks the back button in the nav. In this case they would need to go back to one of 3 places:
a. if they're a cloud customer, the last page would be the select gh server screen
b. if they're a server customer the last page could either be:
i. the manual form screen
ii. the auto/manual selection screen (customer selected auto in this case).

Reply:
I believe this solution works for both scenarios. Heres my understanding of how this works:
- When user follows the cloud path. 
  - In the GH version selection screen(parent window), there is a hidden input valued null.
  - When the user selects the Cloud option, it will open a new child window.
  - When the user clicks on the connect button of the child window, the child window sets a true value for the hidden input in the parent. 
  - In the parent window, when the child window is closed, it navigates to GH config page only when the value of this hidden input is set to true, which happens only when the connect button is clicked. Otherwise it will just do a normal refresh. 
- When user follows the enterprise path, the user navigates through the same window until he/she reaches the automatic/manual app creation step.
  - If its manual, its still the same window, so nothing changes. 
    - We can set the navigation logic at the successful submission of the manual app form.
  - But if its automatic, it will open a child window. So, if we add a hidden input of the same id, then the rest should function similarly as in the step for the cloud above.
    - However, this input file hasn't been added here for the enterprise server path because the action for the automatic button hasn't been implemented yet.

Reply:
I'm not sure I follow, this file is only used in the jira select screen and as far as I can tell, anytime the window closes, we want to show the landing page no matter what.

maybe we need to get together tomorrow and figure this out over a screenshare.

Reply:
either way, even if we need a way to communicate back to the opener window, there's a better way of doing it, but first let's assess if it's needed at all.

Reply:
- The condition has been removed.
- Updated the logic from a page refresh to a page redirect to GH config Page